.github/ISSUE_TEMPLATE/patternfly-issue-template.md

```diff
@@ -0,0 +1,17 @@
+---
+name: PatternFly issue template
+about: Please populate the PatternFly issue template so we can learn more about your
+  needs.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### Describe the issue. What are the expected and unexpected behavior categories?
+
+### Please provide the steps to reproduce so we can better understand and address the issue. Feel free to link CodePen or another tool.
+
+### Would you consider this issue a bug or enhancement?
```

Comment:
I think this question should be asked first and read like "Is this a bug or feature request?"

Reply:
@redallen we chose this wording because we felt it was most helpful for users who may not know what a feature request is. cc @tlabaj @dgutride

Reply:
Maybe "Is this a bug or an enhancement?"